src/utils/AxiosFactory.ts

```diff
@@ -30,22 +30,32 @@ export default class AxiosFactory {
     if (!axiosInstance) {
       // Create
       axiosInstance = axios.create(instanceConfiguration.axiosConfig);
-      // Add a Request interceptor
-      axiosInstance.interceptors.request.use((request: AxiosRequestConfig) => {
-        Logging.logAxiosRequest(tenantID, request);
-        return request;
-      }, async (error: AxiosError) => {
-        Logging.logAxiosError(tenantID, error);
-        return Promise.reject(error);
-      });
-      // Add a Response interceptor
-      axiosInstance.interceptors.response.use((response: AxiosResponse) => {
-        Logging.logAxiosResponse(tenantID, response);
-        return response;
-      }, async (error: AxiosError) => {
-        Logging.logAxiosError(tenantID, error);
-        return Promise.reject(error);
-      });
+
+      // FIXME
+      // Error Message: Converting circular structure to JSON
+      // Temporary FIX: Add no inceptors
+      if (!noInterceptors) {
```

Comment:
That parameter is optional, if not specified, you have to default it a true value.

Reply:
ok, I thought that if the the optional parameter is undefined the **if (!noInterceptors)** result would be true